src/results/outages.jl

```diff
@@ -67,13 +74,24 @@ function add_outage_results(m, p, d::Dict)
 	r["unserved_load_per_outage_kwh"] = round.(unserved_load_per_outage, digits=2)
 	r["storage_microgrid_upgrade_cost"] = value(m[:dvMGStorageUpgradeCost])
 	r["microgrid_upgrade_capital_cost"] = r["storage_microgrid_upgrade_cost"]
+    _n = ""
```

Comment:
If multinode doesn't work with this multiple outage modeling, why bother with doing this and then possibly giving the illusion that they are compatible, rather than just doing m[:dvStorageEnergy]?

Reply:
Sure, address in d24c2e2b348ce9d6245913b25a4d95854f9df48c.